src/main/java/com/internet/shop/dao/jdbc/CarDaoJdbcImpl.java

```diff
@@ -0,0 +1,222 @@
+        Long carId = car.getId();
+        try (Connection connection = ConnectionUtil.getConnection();
+                PreparedStatement updateCars = connection.prepareStatement(updateQuery);
+                PreparedStatement deleteDrivers =
+                        connection.prepareStatement(deleteQuery);
+                PreparedStatement insertDrivers =
+                        connection.prepareStatement(insertQuery)) {
+            updateCars.setLong(1, car.getManufacturer().getId());
+            updateCars.setString(2, car.getModel());
+            updateCars.setLong(3, carId);
+            updateCars.executeUpdate();
+            updateCars.close();
+            deleteDrivers.setLong(1, car.getId());
+            deleteDrivers.executeUpdate();
+            deleteDrivers.close();
+            for (Driver driver : car.getDrivers()) {
+                insertDrivers.setLong(1, driver.getId());
+                insertDrivers.setLong(2, car.getId());
+                insertDrivers.executeUpdate();
+                insertDrivers.close();
+            }
+            return car;
+        } catch (SQLException e) {
+            throw new DataProcessingException("We can`t update this car:" + car, e);
+        }
+    }
+
+    @Override
+    public boolean delete(Long id) {
+        String deleteQuery = "UPDATE cars SET deleted = ?  WHERE car_id = ?";
+        try (Connection connection = ConnectionUtil.getConnection();
+                PreparedStatement statement = connection.prepareStatement(deleteQuery)) {
+            statement.setBoolean(1, true);
+            statement.setLong(2, id);
+            return statement.executeUpdate() > 0;
+        } catch (SQLException e) {
+            throw new DataProcessingException("Can`t delete data with id:"
+                    + id + " from DB ", e);
+        }
+    }
+
+    @Override
+    public List<Car> getAll() {
+        String getAllQuery = "SELECT car_id, car_model, m.manufacture_id, "
+                + "m.manufacture_name, m.manufacture_country "
+                + "FROM cars c "
+                + "JOIN manufacturers m ON c.manufacturer_id = m.manufacture_id "
+                + "WHERE c.deleted = FALSE;";
+        List<Car> allCars = new ArrayList<>();
+        try (Connection connection = ConnectionUtil.getConnection();
+                PreparedStatement statement = connection.prepareStatement(getAllQuery)) {
+            ResultSet resultSet = statement.executeQuery(getAllQuery);
+            while (resultSet.next()) {
+                allCars.add(createCar(resultSet));
+            }
+        } catch (SQLException e) {
+            throw new RuntimeException("Can`t get all data from DB ", e);
+        }
+        for (Car car : allCars) {
+            car.setDrivers(getDrivers(car.getId()));
+        }
+        return allCars;
+    }
+
+    @Override
+    public List<Car> getAllByDriver(Long id) {
+        String getAllByDriverQuery = "SELECT d.driver_id, d.driver_name, d.license_number, "
```

Comment:
```suggestion
        String getAllByDriverQuery = "SELECT  "
```
do we need to select these columns? do we use them?